webapp/packages/core-ui/src/InlineEditor/InlineEditor.tsx

```diff
@@ -28,6 +39,7 @@ export interface InlineEditorProps extends Omit<React.InputHTMLAttributes<HTMLIn
   autofocus?: boolean;
   active?: boolean;
   loading?: boolean;
+  autoCompleteProposals?: InputAutocompleteProposal[];
```

Comment:
please do not extend `InlineEditor` functionality cuz its deprecated and we want to use our base controls instead